### ISSUE TITLE
When we can expect the database to be back?

### ISSUE THREAD

**Issue by Wlada** [shakee93/fonoapi#77]
When we can expect the database to be back? Thanks

**Comment by maheshperi19** [comment]
Yes, we are eagerly waiting for the update with images. Better to have all phones images along with the specifications.

**Comment by Wlada** [comment]
It's not just images, I can't get basic phone data any more.

**Comment by piotrbarandeveloper** [comment]
@shakee93  well?

**Comment by NeleB54Gold** [comment]
You could put the database open source to avoid load issues, there are no other alternatives like this

**Comment by wobsoriano** [comment]
I can scrape GSMARENA site for all phones and use Vercel serverless for hosting API and maybe faunadb as database

**Comment by NeleB54Gold** [comment]
> I can scrape GSMARENA site for all phones and use Vercel serverless for hosting API and maybe faunadb as database 

If you can create and make open source it, so you can do a self host, you wouldn't have heavy load issues like fonoapi, right? Would be great

**Comment by umutesen** [comment]
[this](https://api.deviceinfo.xyz) appears to be a similar API available.

**Comment by wobsoriano** [comment]
This project is dead, they are just shy to admit it. Good run tho.

**Comment by leshawn-rice** [comment]
I am working on a similar API at the moment. It should be finished within the next few weeks. [Github Link](https://github.com/leshawn-rice/grabaphone)

**Comment by joehanna** [comment]
> [this](https://api.deviceinfo.xyz) appears to be a similar API available.

This returns a 503 error now

**Comment by jamureddy** [comment]
Hi ,we have an api similar to fono api

We are launching our api very soon ,if intersted check this url.

**https://api.teksitus.com/launch/.**

Free for developer and personal projects.

Teksitus

**Comment by jamureddy** [comment]
> When we can expect the database to be back? Thanks
Hi ,we have an api similar to fono api

We are launching our api very soon ,if intersted check this url.

https://api.teksitus.com/launch/.

Free for developer and personal projects.

Teksitus

**Comment by jamureddy** [comment]
> > [this](https://api.deviceinfo.xyz) appears to be a similar API available.
> 
> This returns a 503 error now

Hi ,we have an api similar to fono api

We are launching our api very soon ,if intersted check this url.

https://api.teksitus.com/launch/.

Free for developer and personal projects.

Teksitus

**Comment by maheshperi19** [comment]
> > > [this](https://api.deviceinfo.xyz) appears to be a similar API available.
> > 
> > 
> > This returns a 503 error now
> 
> Hi ,we have an api similar to fono api
> 
> We are launching our api very soon ,if intersted check this url.
> 
> https://api.teksitus.com/launch/.
> 
> Free for developer and personal projects.
> 
> Teksitus

When can we expect this new api launch ?

**Comment by realtechspecs** [comment]
We're launching an official API for the world's consumer electronics tech specs. 

Our API is ready and currently in private beta. 

You can join the waitlist here https://developer.techspecs.io/ and try our search engine here https://techspecs.io/ (powered by our API)
